tests/generate_source/build.bp

```diff
@@ -189,12 +189,38 @@ bob_binary {
         "generate_source_single_dependend",
         "generate_source_single_dependend_nested",
     ],
+    generated_headers: [
+        "generate_source_single",
+    ],
     srcs: ["main.cpp"],
 }
 
+bob_generate_source {
+    name: "gen_sources_and_headers",
+    srcs: [
+        "lib.c.txt",
+        "lib.h.txt",
+    ],
+    out: [
+        "foo/src/foo.c",
+        "foo/foo.h",
+    ],
+    export_gen_include_dirs: ["."],
+    tool: "gen.sh",
+    cmd: "${tool} ${gen_dir} ${in}"
+}
+
+bob_binary {
+    name: "foo_generated",
```

Comment:
Pref: Can you rename this too please? Same reasoning as gen_sources_and_headers.